api/src/controllers/user.js

```diff
@@ -699,6 +699,10 @@ router.get(
         termsAccepted: user.termsAccepted,
         cgusAccepted: user.cgusAccepted,
         gaveFeedbackEarly2023: user.gaveFeedbackEarly2023,
+        lastLoginAt: user.lastLoginAt,
+        decryptAttempts: user.decryptAttempts,
+        disabledAt: user.disabledAt,
+        loginAttempts: user.loginAttempts,
```

Comment:
Je suggère de ne pas renvoyer ces données là, vu qu'on n'en a pas besoin en tant qu'utilisateur et que ce n'est pas lié. Ça donne une nouvelle info dont on n'a pas besoin, on verra si un jour on leak cette info ou pas


```suggestion
        lastLoginAt: user.lastLoginAt,
        disabledAt: user.disabledAt,
```

Reply:
on a ailleurs dans le code

```
export const usersLastLoginMoreThan6MonthsSelector = selector<number>({
  key: "usersLastLoginMoreThan6MonthsSelector",
  get: ({ get }) => {
    const users = get(usersState);
    return users.filter((user) => dayjs().diff(user.lastLoginAt ?? user.createdAt, "months") > 6).length;
  },
});

export const usersTooManyDecryptAttempsSelector = selector<number>({
  key: "usersTooManyDecryptAttempsSelector",
  get: ({ get }) => {
    const users = get(usersState);
    return users.filter((user) => user?.decryptAttempts > 12).length;
  },
});
```

Reply:
du coup je suis plus pour renvoyer `decryptAttempts` au moins

Reply:
pour être honnête je n'ai pas réfléchi, j'ai copié collé ce qu'on renvoie par ailleurs dans le `get /users`